calypso/csadmin/clicontracts/lts_test.sh

```diff
@@ -0,0 +1,50 @@
+# This method should be called from the calypso/csadmin/test.sh script
+
+testContractLTS() {
+    run testContractLTSInvoke
```

Comment:
why do we need an extra wrapper? can we not just do `run testContractLTSInvoke` inside `test.sh`?

Reply:
Yes we could, this is in anticipation if more functions are added to this clicontract (for example `get` or `delete`). 

It comes from what has been done in `byzcoin/bcadmin/clicontracts`.